membership/interface.go

```diff
@@ -0,0 +1,14 @@
+package membership
+
+// Member defines a member of the membership that can be used by the rest of the
```

Comment:
What does "used by the rest of the system" mean? Isn't is just a definition of a member in the membership?